src/components/JSONEditor.tsx

```diff
@@ -39,9 +24,11 @@ export default function JSONEditor({
   ) : (
     <CodeMirror
       value={value}
-      autoCursor={false}
       onChange={onCodeChange}
-      options={cmOptions}
+      height="auto"
+      style={{}}
+      extensions={[langs.json()]}
+      indentWithTab={true}
```

Comment:
We had `indentWithTabs: false,` with options like `tabSize: 2,`. Why this change?